asset/css/search-base.less

```diff
@@ -264,6 +276,10 @@ fieldset:disabled .term-input-area [data-drag-initiator] {
       }
     }
   }
+
+  &.read-only [data-index] > .remove-action {
```

Comment:
Please do not set a higher specificity here. It should be the same as the rule for non-vertical. So please remove `>`.